From b7be69d43411b190df48a11f69ea87009d1603ac Mon Sep 17 00:00:00 2001
From: Atanas Bunchev <atanas.bunchev@konsulko.com>
Date: Tue, 28 Nov 2023 20:43:10 +0200
Subject: [PATCH 2/2] ti_armv7_common.env: Increase allocated space for kernel

The uncompressed kernel gets too big after adding the Balena-specific modifications.

Signed-off-by: Atanas Bunchev <atanas.bunchev@konsulko.com>
---
 include/environment/ti/ti_armv7_common.env | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/include/environment/ti/ti_armv7_common.env b/include/environment/ti/ti_armv7_common.env
index e87a41a659..b4017112c8 100644
--- a/include/environment/ti/ti_armv7_common.env
+++ b/include/environment/ti/ti_armv7_common.env
@@ -1,18 +1,18 @@
 loadaddr=0x82000000
 kernel_addr_r=0x82000000
-fdtaddr=0x88000000
-dtboaddr=0x89000000
-fdt_addr_r=0x88000000
-fdtoverlay_addr_r=0x89000000
-rdaddr=0x88080000
-ramdisk_addr_r=0x88080000
+fdtaddr=0x89000000
+dtboaddr=0x90000000
+fdt_addr_r=0x89000000
+fdtoverlay_addr_r=0x90000000
+rdaddr=0x89080000
+ramdisk_addr_r=0x89080000
 scriptaddr=0x80000000
 pxefile_addr_r=0x80100000
 bootm_size=0x10000000
 boot_fdt=try
 
 boot_fit=0
-addr_fit=0x90000000
+addr_fit=0x91000000
 name_fit=fitImage
 update_to_fit=setenv loadaddr ${addr_fit}; setenv bootfile ${name_fit}
 get_overlaystring=
-- 
2.39.2

